code/mob.dm

```diff
@@ -12,7 +12,8 @@
 
 	var/datum/abilityHolder/abilityHolder = null
 	var/datum/bioHolder/bioHolder = null
-	var/datum/targetable/targeting_spell = null
+
+	var/targeting_ability = null
```

Comment:
Is this no longer a datum? Might want to type it.

Reply:
Think of it as an union type for /datum/targetable and /obj/ability_button, ZeWaka! I'll do some refactoring to give it a proper type after we merge this.

Reply:
zewaka its no longer typed bc i was trying to stay away from adding more vars to mob. the only code that relied on it being typed im pretty sure i fixed with an istype